.github/guides/STANDARDS.md

```diff
@@ -256,9 +256,10 @@ Good:
 #define OUR_ON_OVERLAY 1
 #define OUR_OFF_OVERLAY 2
 #define OUR_BROKEN_OVERLAY 3
-/obj/machine/update_overlays(var/blah
+
+/obj/machine/update_overlays(var/blah)
 	var/static/our_overlays
-	if(isnull(our_overlays)
+	if (isnull(our_overlays))
```

Comment:
I get the indentation but is `if()` really worse then `if ()`?

Reply:
As long as we're not enforcing it across the codebase, I don't care about this specific change.

By and large `if()` is the most predominantely used variant in our codebase and I'd assume that `if()` is the preferred style as a result. There are about ~68,000 matches for `if()`, versus ~6,400 for `if ()`.

I see this as being similar to British English vs. American English. It doesn't matter which you use. It doesn't aid code readbility nor make the code easier to understand and the compiler we use treats `if()` and `if ()` the same. As such, I see it as personal preference.

Reply:
> 

https://github.com/tgstation/tgstation/blob/master/.github/guides/STYLE.md#things-that-do-not-matter

Reply:
Formally we don't care, I was just changing it to be more consistent with the other code *in this sample*.

Reply:
Aight